doc/mkdocs/docs/api/basic_json/operator[].md

```diff
@@ -6,12 +6,16 @@ reference operator[](size_type idx);
 const_reference operator[](size_type idx) const;
 
 // (2)
-reference operator[](const typename object_t::key_type& key);
-const_reference operator[](const typename object_t::key_type& key) const;
+template<typename KeyT>
+reference operator[](KeyT && key);
+template<typename KeyT>
+const_reference operator[](KeyT && key) const;
 template<typename T>
 reference operator[](T* key);
 template<typename T>
 const_reference operator[](T* key) const;
+reference operator[](const std::string_view& key); // since C++17
+const_reference operator[](const std::string_view& key) const; // since C++17
```

Comment:
I believe that the template function is sufficient to cover the string_view, and T* case as well.

Reply:
Good point. I'll refactor.